Pilot/DirectoryService.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  DirectoryService.swift
+//  Pilot
+//
+//  Created by Nick Eckert on 6/28/16.
+//  Copyright © 2016 Sanction. All rights reserved.
+//
+
+import Cocoa
+import FileKit
+
+class DirectoryService: NSObject {
+
+  static func getFilesFromPath(platformType: PlatformType, caller: FileService) -> [LocalFile]? {
```

Comment:
I chose nil because if a problem occurs and an empty list is returned then a sync would attempt to download all the cloud files regardless of wether or not they exist. With nil I know a problem occurred and I can handle it accordingly